multiqc/modules/mosdepth/mosdepth.py

```diff
@@ -197,24 +212,36 @@ def parse_cov_dist(self):
             assert type(exclude_contigs) == list, type(exclude_contigs)
         except (AttributeError, TypeError, AssertionError):
             exclude_contigs = []
-        log.debug("include_contigs: {}".format(include_contigs))
-        log.debug("exclude_contigs: {}".format(exclude_contigs))
-
+        log.debug(f"include_contigs: {include_contigs}")
+        log.debug(f"exclude_contigs: {exclude_contigs}")
+
+        # Parse mean coverage
+        for f in self.find_log_files("mosdepth/summary"):
+            s_name = self.clean_s_name(f["fn"], f).replace(".mosdepth", "")
```

Comment:
Better to do this with `config.fn_clean_exts` so that it can be configured.

```suggestion
            s_name = self.clean_s_name(f["fn"], f)
```